### PR TITLE
added new section "api reference" with definitions of all our APIs

### PR DESCRIPTION
**Pull request by edvinasmuncelis**

### CODE REVIEW

File: openapi/payments_api.yaml [hunk 1]
```diff
@@ -0,0 +1,535 @@
...
+            items:
+              enum:
+                - initiated
+                - reserved
+                - captured
+                - cancelledByMerchant
+                - captureStarted
+                - cancelledByUser
+                - cancelledBySystem
+                - terminated
```
Comment (granatoidas) [inline openapi/payments_api.yaml:97]: terminated and captureStarted were not present before

File: openapi/webhooks.yaml [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+            application/json:
+              schema:
+                $ref: '#/components/schemas/SingleWebhookResponse'
+        '400':
+          description: Bad Request
+        '401':
+          description: Unauthorized
+        '409':
+          description: Conflict
+  /api/v1/webhooks/{webhookid}:
```
Comment (granatoidas) [inline openapi/webhooks.yaml:48]: remove /api

File: openapi/payments_api.yaml [hunk 3]
```diff
@@ -0,0 +1,535 @@
...
+      operationId: simulate-confirmation
+      description: Simulate user swiping and confirming the payment (**available only in sandbox environment**).
+      servers:
+        - url: https://api.sandbox.mobilepay.dk
+          description: This endpoint is available only in sandbox environment
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/CancelPaymentRequest'
```
Comment (granatoidas) [inline openapi/payments_api.yaml:267]: Seems like integraton-test/reserve should have a different requestBody

File: openapi/payments_api.yaml (hunk 1 above)
Comment (edvinasmuncelis) [reply openapi/payments_api.yaml:97]: removed

File: openapi/payment_points.yaml [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+        paymentPointName:
+          type: string
+          description: Merchant-defined Payment Point name.
+          nullable: true
+          example: MyPaymentPoint
+        state:
+          enum:
+            - created
+            - active
+            - deleted
```
Comment (granatoidas) [inline openapi/payment_points.yaml:120]: this state should not be here

File: openapi/payments_api.yaml (hunk 3 above)
Comment (edvinasmuncelis) [reply openapi/payments_api.yaml:267]: fixed

File: openapi/payment_points.yaml (hunk 4 above)
Comment (edvinasmuncelis) [reply openapi/payment_points.yaml:120]: fixed

**Review by granatoidas — APPROVED** [review]

**Pull request merged**